common/src/main/java/org/apache/atlas/repository/Constants.java

```diff
@@ -324,6 +324,9 @@ public enum SupportedFileExtensions { XLSX, XLS, CSV }
 
     public static final int ELASTICSEARCH_PAGINATION_SIZE = 50;
 
+    public static final String ENTITY_DELETED_STATUS    = "DELETED";
```

Comment:
This is not needed
Use AtlasEntity.Status.DELETED.name() instead

Reply:
Fixed it check it now